README.md

```diff
@@ -46,27 +43,26 @@ workflows:
   build:
     jobs:
       - cypress/run # "run" job comes from "Cypress" orb
+          start-command: 'npm run start'
 ```
 
 By using the `run` job definitions that Cypress provides inside the orb, it
 brings simplicity and static checks of parameters to your CircleCI
 configuration.
 
-You can find multiple examples at
+You can find more usage examples at
 [our official orb page](https://circleci.com/developer/orbs/orb/cypress-io/cypress).
 
-### Arguments
+#### Arguments
 
 You can pass arguments to the `cypress/run` job to override any default behaviors. You can find the full list of arguments at [https://circleci.com/developer/orbs/orb/cypress-io/cypress#jobs-run](https://circleci.com/developer/orbs/orb/cypress-io/cypress#jobs-run)
 
 ## Parallelization
 
-A more complex project that needs to install dependencies, build an application
-and run tests across 4 CI machines [in parallel](/guides/guides/parallelization)
+A more complex project that needs to install dependencies 
+and run tests across 4 CI machines [in parallel](https://docs.cypress.io/guides/guides/parallelization)
 may have:
 
-> Note: `--parallel` flag tells Cypress to run tests in parallel. `parallelism` parameter tells Circle to run across `n` number of machines.
```

Comment:
Do you feel like this isn't important to call out here now?

Reply:
Yeah I put back the comments in the code block below, which covers the same stuff. Felt unnecessary with both.